routers/api/actions/runner/utils.go

```diff
@@ -0,0 +1,221 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
```

Comment:
I'm not sure if this file is actually needed - in the [original PR](https://github.com/go-gitea/gitea/pull/25664/files#diff-cd4aa1f217c85f6ea341104754c4ad93b3526a2222cd14b8011180f3343c5462) there was just a small change to its logic, but it looks like the file has been [entirely deleted in the main branch of the repo](https://github.com/go-gitea/gitea/tree/5407382b43bb73e25e9ecd77e74941ac38d37d17/routers/api/actions/runner). I _think_ it can be deleted entirely, but I'll keep it around until either a) I can get more confidence in that position, or b) someone more knowledgeable about the codebase can confirm that.

Reply:
It has been moved in a recent PR. Maybe you could try to find when it is deleted. https://stackoverflow.com/questions/6839398/find-when-a-file-was-deleted-in-git

Reply:
Ah, thanks! Looks like it was [moved](https://github.com/go-gitea/gitea/commit/da33b708af1a98c2ead4b4714fcc3055b2d4c6bd) to [services/actions/task.go](https://github.com/go-gitea/gitea/blob/main/services/actions/task.go).